CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
-## 0.7.13-dev1
+## 0.7.13-dev2
```

Comment:
let's make this a non-dev release so it can get integrated into unstructured sooner rather than later.